### DISCUSSION TITLE
API to use calendar from other plugins

### DISCUSSION THREAD

**Discussion by sandorex** [davish/obsidian-full-calendar#379] [Feature Requests]
I would like if there was a stable api to add events and such from other plugins, then it could be extended to any format in separate plugins however exotic it is

My particular use case is to add inline parsing of task begin/end dates

**Comment by davish** [comment]
I am hoping this will be completed as part of the refactor in #204!

**Comment by mayurankv** [comment]
Now that #204 has been merged is this something that is currently available?

**Comment by davish** [comment]
#204 ended up not linking up the new code with the view layer. So there is
not yet any API that can be hooked into, since the new cache is not
initialized anywhere.

**Comment by mayurankv** [comment]
Understood, thanks for clarifying.
